app/models/post.rb

```diff
@@ -27,6 +27,9 @@ class Post < ApplicationRecord
   has_many :content_warnings, -> { ordered_by_post_tag }, through: :post_tags, source: :content_warning,
     after_add: :reset_warnings, dependent: :destroy
 
+  has_many :post_fonts, dependent: :destroy
+  has_many :fonts, through: :post_fonts, dependent: :destroy
```

Comment:
Add to a unit test somewhere ensuring this destroy happens? (to the PostFont and not the Font)